src/main/scala/beam/router/model/EmbodiedBeamTrip.scala

```diff
@@ -126,6 +132,8 @@ object EmbodiedBeamTrip {
       DRIVE_TRANSIT
     } else if (theMode == TRANSIT && hasUsedBike) {
       BIKE_TRANSIT
+    } else if (theMode == TRANSIT && hasUsedEMERGENCY) {
+      EMERGENCY_TRANSIT
```

Comment:
I think that it's safier not to change these method at all. But in this case we wouldn't have `EMERGENCY` in `PathTraversalEvent`.